### ISSUE TITLE
[Test Issue] [OpcacheOn] Opcache/opt test case fail when opcache.opt_debug_level=0x20000

### ISSUE THREAD

**Issue by hollyhuiLi** [php/pftt2#62]
among all the opt test cases, 26 test cases having debug level 0x020000 all failed.

ext/opcache/tests/opt/dce_001.phpt
ext/opcache/tests/opt/dce_002.phpt
ext/opcache/tests/opt/dce_003.phpt
ext/opcache/tests/opt/dce_004.phpt
ext/opcache/tests/opt/dce_005.phpt
ext/opcache/tests/opt/dce_006.phpt
ext/opcache/tests/opt/dce_007.phpt
ext/opcache/tests/opt/dce_008.phpt
ext/opcache/tests/opt/jmp_001.phpt
ext/opcache/tests/opt/sccp_001.phpt
ext/opcache/tests/opt/sccp_002.phpt
ext/opcache/tests/opt/sccp_003.phpt
ext/opcache/tests/opt/sccp_004.phpt
ext/opcache/tests/opt/sccp_005.phpt
ext/opcache/tests/opt/sccp_006.phpt
ext/opcache/tests/opt/sccp_007.phpt
ext/opcache/tests/opt/sccp_008.phpt
ext/opcache/tests/opt/sccp_009.phpt
ext/opcache/tests/opt/sccp_010.phpt
ext/opcache/tests/opt/sccp_011.phpt
ext/opcache/tests/opt/sccp_012.phpt
ext/opcache/tests/opt/sccp_016.phpt
ext/opcache/tests/opt/sccp_017.phpt
ext/opcache/tests/opt/sccp_019.phpt
ext/opcache/tests/opt/sccp_022.phpt
ext/opcache/tests/opt/sccp_024.phpt

**Comment by cmb69** [comment]
I can reproduce this (Windows only) with run-test.php, so this is not directly related to PFTT, but rather a general issue. Please file a bug report in the [bugtracker](https://bugs.php.net/), and close this ticket afterwards. Thanks.

**Comment by cmb69** [comment]
Thanks! So I'll have a closer look ASAP.

**Comment by cmb69** [comment]
You're right. The problem is that these opcache dumps are written to stderr, but apparently stderr is ignored by the test output reader.

**Comment by cmb69** [comment]
> […] but apparently stderr is ignored by the test output reader.

Hmm, not always at least (a [respective comment](https://github.com/php/pftt2/blob/1354c8c0f18ed9a49f3fd353d8a6f823131ea7b6/src/com/mostc/pftt/host/LocalHost.java#L474) is confusing). When debugging, sometimes the tests passed for me. Also setting `opcache.log_verbosity_level` to 2 (instead of 0) and triggering an OPcache warning (e.g. by setting `opcache.file_cache` to a non existing folder) makes the tests pass (unless `opcache.error_log` is set). Looks like some stderr output is ignored, but I don't know why.
